server/src/main/java/ru/practicum/shareit/booking/service/BookingServiceImpl.java

```diff
@@ -142,72 +145,127 @@ public List<BookingDto> getBooking(BookingStatus state, Integer bookerId, Intege
+//        } else {
+//            throw new UserNotFoundException("Пользователь не найден");
+//        }
+//    }
+//
+
+    public List<BookingDto> ownerItemsBookingLists(String state, Integer userId, Integer from, Integer size) {
+        List<Booking> list;
+        if (from == null && size == null) {
+            Optional<User> user = userRepository.findById(userId);
+            if (user.isPresent()) {
+                List<Booking> listByOwner = bookingRepository.findAllByItemOwnerId(userId);
+                list = getBookingListByStatus(state, listByOwner);
+            } else {
+                throw new UserNotFoundException("Пользователь не найден");
+            }
+        } else if (from >= 0 && size > 0) {
+            Pageable pageable = PageRequest.of(from / size, size, Sort.by(Sort.Direction.DESC, "dateBegin"));
+            log.info("*****************************************************");
+            switch (state) {
+                case "PAST":
+                    list = bookingRepository.findAllByItemOwnerIdAndDateEndBefore(userId, LocalDateTime.now(),
+                            pageable).getContent();
+                    log.info("lsit: " + list + "  " + "PAST");
+                    break;
+                case "FUTURE":
+                    list = bookingRepository.findAllByItemOwnerIdAndDateBeginAfter(userId, LocalDateTime.now(),
+                            pageable).getContent();
+                    log.info("lsit: " + list + "  " + "FUTURE");
+                    break;
+                case "CURRENT":
+                    list = bookingRepository.findAllByItemOwnerIdAndDateBeginBeforeAndDateEndAfter(userId,
+                            LocalDateTime.now(), LocalDateTime.now(), pageable).getContent();
+                    break;
+                case "WAITING":
+                    list = bookingRepository.findAllByItemOwnerIdAndStatus(userId, "WAITING", pageable).getContent();
+                    log.info("lsit: " + list + "  " + "WAITING");
+                    break;
+                case "REJECTED":
+                    list = bookingRepository.findAllByItemOwnerIdAndStatus(userId, "REJECTED", pageable).getContent();
+                    log.info("lsit: " + list + "  " + "REJECTED");
+                    break;
+                case "ALL":
+                    list = bookingRepository.findAllByItemOwnerId(userId, pageable).getContent();
+                    log.info("lsit: " + list + "  " + "ALL");
+                    break;
+                default:
+                    throw new BookingUnknownStateException(state);
+            }
+            log.info("*****************************************************");
         } else {
-            throw new UserNotFoundException("Пользователь не найден");
+            throw new IncorrectBookingParameterException("Неверные параметры");
         }
+        return list.stream().map(BookingMapper::toBookingDto).collect(Collectors.toList());
```

Comment:
Стримы принято бить по строкам для лучшей читаемости кода
```suggestion
        return list.stream()
            .map(BookingMapper::toBookingDto)
            .collect(Collectors.toList());
```